### DISCUSSION TITLE
TypeError: 'bool' object is not callable in Computer vision training code

### DISCUSSION THREAD

**Discussion by The-Ali-Sher** [mrdbourke/pytorch-deep-learning#864] [Q&A]
Video: https://www.youtube.com/watch?v=Z_ikDlimN6A&t=54634s 03 pytorch computer vision

I'm getting this error when running the training and test loop code:
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
[<ipython-input-47-97ae4d91263a>](https://localhost:8080/#) in <cell line: 12>()
     45     model_0.eval()
     46     with torch.inference_mode():
---> 47         for X, y in test_dataloader:
     48             # 1. Forward pass
     49             test_pred = model_0(X)

4 frames
[/usr/local/lib/python3.10/dist-packages/torchvision/datasets/mnist.py](https://localhost:8080/#) in __getitem__(self, index)
    146 
    147         if self.target_transform is not None:
--> 148             target = self.target_transform(target)
    149 
    150         return img, target

TypeError: 'bool' object is not callableI even tried copying the code directly from learnpytorch.io but still getting the same error

**Comment by LuluW8071** [comment]
How can we help when there is no snippet of your code.
We can't debug just by knowing the error😐
